styles.css

```diff
@@ -197,6 +197,243 @@ header {
   font-size: 1.2rem;
 }
 
+.about-myself {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  background-color: white;
+  border-top-right-radius: 80px;
+  padding: 30px 0;
+  margin-top: 50px;
+}
+
+.social-media ul {
+  list-style: none;
+  flex-direction: row;
+  display: flex;
+  padding-left: 0;
+  margin-left: -15px;
+}
+
+.social-media a {
+  margin: 0 10px;
+  color: black;
+  width: auto;
+}
+
+.resume {
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 6px;
+  padding: 15px 10px;
+  background-color: white;
+}
+
+.resume:hover {
+  background-color: #6070ff;
+  color: white;
+  cursor: pointer;
+}
+
+.contacts {
+  display: flex;
+  align-items: center;
+  background-color: #6070ff;
+  color: #ebebff;
+  border-top-left-radius: 80px;
+}
+
+.web-languages-card {
+  margin: 5px;
+  padding: 20px;
+  padding-top: 20px;
+  padding-left: 10px;
+  background: #f7f7f9;
+  size: 20px;
+}
+
+.languages {
+  display: flex;
+  flex-direction: column;
+}
+
+.web-languages {
+  display: flex;
+}
+
+.about-introduction {
+  color: #091e42;
+  font-weight: bolder;
+  font-size: 30px;
+}
+
+.technologies-link {
+  padding-top: 15px;
+  padding-bottom: 15px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.languages-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+h4 {
+  color: #7f8cff;
+  font-size: 10px;
+  font-weight: 500;
+}
```

Comment:
- [ ] [OPTIONAL] Kindly increase the font size of your text so it doesn't give people with eye defects an issue to see. 🙏 

![image](https://user-images.githubusercontent.com/74814780/176549519-9734c30f-1ea5-4b02-9f91-24ed46bdc932.png)